src/System.Private.CoreLib/src/System/Environment.Unix.cs

```diff
@@ -84,5 +84,10 @@ public static unsafe string MachineName
                 return Encoding.UTF8.GetString(hostName, hostNameLength);
             }
         }
+
+        public static void Exit(int exitCode)
+        {
```

Comment:
I think just stick a TODO here for now. I have a work in progress to get rid of some of these assembly helpers for GC. I will do the same for the shutdown helper as well.